storage/series_test.go

```diff
@@ -126,14 +126,13 @@ func TestChunkSeriesSetToSeriesSet(t *testing.T) {
 }
 
 type histogramTest struct {
-	samples              []tsdbutil.Sample
-	expectedChunks       int
-	expectedCounterReset bool
+	samples               []tsdbutil.Sample
+	expectedCounterResets []chunkenc.CounterResetHeader
```

Comment:
Or maybe `expCounterResetHeaders`
```suggestion
	expectedCounterResetHeaders []chunkenc.CounterResetHeader
```